runtime/litmus/src/lib.rs

```diff
@@ -415,10 +415,27 @@ impl pallet_scheduler::Config for Runtime {
 	// otherwise I get: panicked at 'called `Result::unwrap()` on an `Err` value: ()'
 	type WeightInfo = ();
 	type OriginPrivilegeCmp = frame_support::traits::EqualPrivilegeOnly;
-	type PreimageProvider = ();
+	type PreimageProvider = Preimage;
 	type NoPreimagePostponement = ();
 }
 
+parameter_types! {
+	pub const PreimageMaxSize: u32 = 4096 * 1024;
+	pub const PreimageBaseDeposit: Balance = 1 * DOLLARS;
+	// One cent: $10,000 / MB
+	pub const PreimageByteDepositBalance: Balance = 1 * CENTS;
```

Comment:
We already have a PreimageByteDeposit defined in pallet_democracy, shall we simply use that?
`type ByteDeposit = PreimageByteDeposit;`
That seems to be what most of the teams do.

Reply:
My idea is a new configure to make it possible to set a different value. Let's follow them if most of the projects implement like this way. will update with new commit

Reply:
yea let's first set it this way and see how it goes